docs/user-guide.md

```diff
@@ -213,18 +204,22 @@ Analyzes the project for a list of its dependencies, optionally uploading the re
 ```
 
-| Flag            | Short | Description                                                                  |
-| --------------- | ----- | ---------------------------------------------------------------------------- |
-| `--config`      | `-c`  | Path to a [configuration file](/docs/config-file.md) including filename.     |
-| `--project`     | `-p`  | Configuration value for [project](/docs/config-file.md/#project-optional).   |
-| `--revision`    | `-r`  | Configuration value for [revision](/docs/config-file.md/#revision-optional). |
-| `--endpoint`    | `-e`  | Configuration value for [endpoint](/docs/config-file.md/#endpoint-optional). |
-| `--output`      | `-o`  | Output `fossa analyze` results to stdout.                                    |
-| `--team`        | `-T`  | Connect this project with the specified existing team in FOSSA.              |
-| `--server-scan` |       | Run a server side dependency scan on raw modules.                            |
-| `--dev`         |       | Include development dependencies. CAUTION: valid only for nodejs projects.   |
-| `--debug`       |       | Print debugging information to stderr.                                       |
-| `--help`        | `-h`  | Print a help message.                                                        |
+| Flag            | Short | Description                                                                             |
+| --------------- | ----- | --------------------------------------------------------------------------------------- |
+| `--config`      | `-c`  | Path to a [configuration file](/docs/config-file.md) including filename.                |
+| `--project`     | `-p`  | Configuration value for [project](/docs/config-file.md/#project-optional).              |
+| `--revision`    | `-r`  | Configuration value for [revision](/docs/config-file.md/#revision-optional).            |
+| `--endpoint`    | `-e`  | Configuration value for [endpoint](/docs/config-file.md/#endpoint-optional).            |
+| `--output`      | `-o`  | Output `fossa analyze` results to stdout.                                               |
+| `--server-scan` |       | Run a server side dependency scan on raw modules.                                       |
+| `--dev`         |       | Include development dependencies. CAUTION: valid only for nodejs projects.              |
+| `--debug`       |       | Print debugging information to stderr.                                                  |
+| `--help`        | `-h`  | Print a help message.                                                                   |
+| `--team`        | `-T`  | Connect the project with the specified team in FOSSA. Valid only on project creation.   |
+| `--policy`      |       | Connect the project with the specified policy in FOSSA. Valid only on project creation. |
+| `--title`       | `-t`  | Set the title that appears in the FOSSA UI. Valid only on project creation              |
```

Comment:
See above comments re: phrasing. I'd rephrase as "Applies only to new projects.".